lib/status/status.go

```diff
@@ -350,6 +350,8 @@ type ClusterServer struct {
 	Status string `json:"status"`
 	// FailedProbes lists all failed probes if the node is not healthy
 	FailedProbes []string `json:"failed_probes,omitempty"`
+	// InfoProbes lists all informational failed probes
+	InfoProbes []string `json:"info_probes,omitempty"`
```

Comment:
See my comment about warning vs info - I feel info is an unnecessary replacement for warnings which are already in.

Reply:
It is not really a replacement, the behavior is different: "warning" probes lead to "degraded" status, "info" probes - don't. I replied to your comment there.